libs/render_delegate/render_delegate.cpp

```diff
@@ -649,10 +649,18 @@ void HdArnoldRenderDelegate::_SetRenderSetting(const TfToken& _key, const VtValu
         if (value.IsHolding<VtStringArray>()) {
             const VtStringArray commandLine = value.UncheckedGet<VtArray<std::string>>();
             for (unsigned int i = 0; i < commandLine.size(); ++i) {
+                // husk argument for output image
                 if (commandLine[i] == "-o" && i < commandLine.size() - 2) {
                     _outputOverride = commandLine[i+1];
                     break;
                 }
+                // husk argument for thread count (#1077)
+                if ((commandLine[i] == "-j" || commandLine[i] == "--threads") 
+                        && i < commandLine.size() - 2) {
+                    // if for some reason the argument value is not a number, atoi should return 0
+                    // which is also the default arnold value. 
+                    AiNodeSetInt(_options, str::threads, std::atoi(commandLine[i+1].c_str()));
+                }
```

Comment:
should we also increment `i` ?

Reply:
good point. Done, for both this argument as well as the -o argument above it